core/lib/server/pub.ts

```diff
@@ -342,6 +344,9 @@ export const getPubs = async (
 					.where("PubsInStages.stageId", "=", props.stageId!)
 			)
 			.$if(Boolean(params.onlyParents), (eb) => eb.where("pubs.parentId", "is", null))
+			// .$if(Boolean(search), (eb) =>
+			// 	eb.where(sql`pubs.values::jsonb::text`, "ilike", `%${search}%`)
+			// )
```

Comment:
this doesn't work—I think I'd like to query `latest_values` but I'm not sure that's exposed at this point. I'm also not sure if this search is quite what we want, or if we only want to be searching on the title of a pub (in which case we wouldn't be able to do this until we have that `isTitle` field). my original idea here was to be able to search all pub values as text just on if the search value was in there at all which might not be performant. but either way, my sql isn't good enough 🥲

Reply:
the context editor storybook only filters on title https://github.com/pubpub/platform/blob/3aa56fccd92ebd1c532ea07042b82d0c507d1d92/packages/context-editor/src/stories/mockUtils.ts#L5-L12